client/coral-admin/src/routes/Community/components/FlaggedUser.js

```diff
@@ -85,7 +85,7 @@ class User extends React.Component {
                 <span className={styles.flaggedByLabel}>
                   {t('community.flags')}({ user.actions.length })
                 </span>:
-                  { user.action_summaries.map(
+                { user.action_summaries.map(
```

Comment:
These were auto-fixed `indent` tweaks from `npm run lint-fix`.
Speaking of formatting and fixing, you may want to consider [**prettier**](http://npm.im/prettier) for optimal code formatting.